tcmur_cmd_handler.c

```diff
@@ -745,6 +745,18 @@ static int tcmur_cmd_handler(struct tcmu_device *dev, struct tcmulib_cmd *cmd)
 
 	track_aio_request_start(rdev);
 
+	/*
+	 * Try to passthrough the cmds to the device.
+	 */
+	if (rhandler->handle_cmd) {
+		ret = handle_passthrough(dev, cmd);
+		if (ret != TCMU_NOT_HANDLED)
```

Comment:
I think this sounds good to me.

For the case where a handler cannot do its own aio (nr_threads > 1) then they can just call async_handle_cmd themselves to schedule execution.

Will try to test tonight and merge tomorrow.